core/pkg/ingress/controller/controller.go

```diff
@@ -676,6 +676,15 @@ func (ic *GenericController) getBackendServers() ([]*ingress.Backend, []*ingress
 
 // GetAuthCertificate ...
 func (ic GenericController) GetAuthCertificate(secretName string) (*resolver.AuthSSLCert, error) {
+	key, err := ic.GetSecret(secretName)
+	if err != nil {
+		return &resolver.AuthSSLCert{}, fmt.Errorf("unexpected error: %v", err)
+	}
+	if key != nil {
+		ic.secretQueue.Enqueue(key)
+	}
+	// Enough time to enqueue the new certificate
+	time.Sleep(5 * time.Second)
```

Comment:
do we need this 5 second sleep or can we rely on the enqueue backing off with each failure? (I'd really prefer to not sleep, we can instead document that the secret must exist, and if it dosn't the controller will throw some events and possibly enter exponential backoff)